app/helpers.php

```diff
@@ -1,6 +1,34 @@
 <?php
 
 /**
+ * Build a CSV from the supplied data.
+ *
+ * @param  array  $data
+ * @return \League\Csv $csv
+ */
+function buildCSV($data)
```

Comment:
Ah, good call. I wasn't actually adding this in my PR, just shifting it around to list function alphabetically. I'll create an issue, cause there's a few functions in there that need to be renamed to follow the standard!